src/Sylius/Bundle/ShopBundle/Resources/views/Checkout/thankYou.html.twig

```diff
@@ -1,38 +1,53 @@
-                        {{ render(url('sylius_shop_order_payment_method_update', {'orderId': order.id} )) }}
-                        {{ 'sylius.ui.payment_ready_to_pay'|trans }}
-                        <a href="{{ path('sylius_shop_order_pay', {'orderId': order.id}) }}" id="sylius-pay-link" class="ui primary button">
-                            {{ 'sylius.ui.pay'|trans }}
-                        </a>
-                    </div>
-                {% endif %}
-                {% if lastPayment.method.instructions is not null %}
-                    <div id="sylius-payment-method-instructions" class="sub header">
-                        {{ lastPayment.method.instructions }}
-                    </div>
-                {% endif %}
+                <div class="ui segment">
+                    {% for payment in order.payments if payment.state is not same as('cancelled')%}
+                        {% if payment.state is same as('new') %}
+                            <i class="circular payment icon"></i>
+                            <div class="ui basic segment">
+                                <div class="ui two column middle aligned very relaxed stackable grid">
+                                    <div class="column left aligned">
+                                        {{ render(url('sylius_shop_order_payment_method_update', {'orderId': order.id} )) }}
+                                    </div>
+                                    <div class="ui vertical divider">
+                                        or
+                                    </div>
+                                    <div class="column">
+                                        <div class="sub header">{{ 'sylius.ui.payment_ready_to_pay'|trans }} </div>
+                                        <div class="ui hidden divider"></div>
+                                        <div class="ui hidden divider"></div>
+                                        <div class="ui hidden divider"></div>
+                                        <a href="{{ path('sylius_shop_order_pay', {'orderId': order.id}) }}" id="sylius-pay-link" class="huge ui primary button">
+                                            {{ 'sylius.ui.pay'|trans }}
+                                        </a>
+                                    </div>
+                                </div>
+                            </div>
+                        {% elseif payment.state is same as('pending') %}
+                            <i class="circular rocket icon"></i>
+                                <div class="ui centered grid">
+                                    <div class="left aligned column">
+                                    {{ render(url('sylius_shop_order_payment_method_update', {'orderId': order.id} )) }}
+                                    </div>
+                                </div>
+                            <div class="ui hidden divider"></div>
+                        {% else %}
```

Comment:
The state that is required to show thank you page (I guess payment should be completed by then).